service/controller/clusterapi/v29/controllercontext/status_types.go

```diff
@@ -113,3 +114,8 @@ type ContextStatusTenantClusterTCNPASG struct {
 	MinSize         int
 	Name            string
 }
+
+type ContextStatusTenantClusterTCNPWorkerInstance struct {
+	DockerVolumeSizeGB string
```

Comment:
This then also needs to be filled in a followup.

Reply:
Yes, I am about to create the `tcnpoutputs` resource.